TWT/apps/challenges/models/challenge.py

```diff
@@ -57,6 +59,13 @@ def __str__(self):
     submissions_status = models.BooleanField(default=False)
     team_creation_status = models.BooleanField(default=False)
     voting_status = models.BooleanField(default=False)
+    youtube_video_link = models.CharField(max_length=43, null=True, blank=True, validators=[
```

Comment:
Why `max_length=43`?

Reply:
because that's the length of youtube links

Reply:
why not models.URLField?

Reply:
wait actually nvm this is fine